project/db/migrate/20221226034219_create_comments.rb

```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string      :contributor, index: true
```

Comment:
@daiphone0228 
[should]
https://aiit2022.slack.com/archives/C04EW39GMCZ/p1672104562350039?thread_ts=1672024253.329929&cid=C04EW39GMCZ

```suggestion
      t.string      :contributor, null: false, index: true
```

Reply:
大変失礼しました。not nullで空文字対応でしたね。
修正しました。